runtimevar/driver/driver.go

```diff
@@ -21,6 +21,17 @@ import (
 	"time"
 )
 
+// DefaultWait is the default value for Wait.
+const DefaultWait = 30 * time.Second
+
+// Wait returns DefaultWait if d is <= 0, otherwise it returns d.
+func Wait(d time.Duration) time.Duration {
```

Comment:
Just this function, or also all of the `Options` struct members too?

Some of them were called `WaitTime` before; I find that weird because it's not a `time.Time`, it's a `time.Duration`. `WaitDuration` might be OK, but it seems a bit chatty for Go.

Reply:
I think `WaitDelay` or `WaitDuration` is OK.

Reply:
OK, done.